tools/mdanalysis/ramachandran_plots.xml

```diff
@@ -0,0 +1,141 @@
+<tool id="mdanalysis_ramachandran_plot" name="Ramachandran Plots" version="0.1.3">
+    <description>Calculate and plot the distribution of two diheadrals in a trajectory</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+<![CDATA[
+    python '$__tool_directory__/ramachandran_plots.py'
+        --idcd '$dcdin' 
+        --ipdb '$pdbin'
+        --isegid1  '$phi.segid1'
+        --iresid1  '$phi.resid1' 
+        --iname1   '$phi.name1'
+        --isegid2  '$phi.segid2'
+        --iresid2  '$phi.resid2' 
+        --iname2   '$phi.name2'
+        --isegid3  '$phi.segid3'
+        --iresid3  '$phi.resid3' 
+        --iname3   '$phi.name3'
+        --isegid4  '$phi.segid4'
+        --iresid4  '$phi.resid4' 
+        --iname4   '$phi.name4'
+        --isegid5  '$psi.segid1'
+        --iresid5  '$psi.resid1' 
+        --iname5   '$psi.name1'
+        --isegid6  '$psi.segid2'
+        --iresid6  '$psi.resid2' 
+        --iname6   '$psi.name2' 
+        --isegid7  '$psi.segid3' 
+        --iresid7  '$psi.resid3' 
+        --iname7   '$psi.name3' 
+        --isegid8  '$psi.segid4' 
+        --iresid8  '$psi.resid4' 
+        --iname8   '$psi.name4' 
+        --output   '$output'  
+        --oramachandran_plot '$ramachandran_plot'
+    2>&1
+]]></command>
+    <inputs>
+        <expand macro="analysis_inputs"/>
+        <section name="phi" title="Phi" expanded="False">
+          <param name="segid1"  type="text" value="HET" label="Segid of atom 1"/>
+          <param name="resid1"  type="text" value="3" label="Resid of atom 1"/>
+          <param name="name1"  type="text" value="O5" label="Atom name of atom 1"/>
+          <param name="segid2"  type="text" value="HET" label="Segid of atom 2"/>
+          <param name="resid2"  type="text" value="3" label="Resid of atom 2"/>
+          <param name="name2"  type="text" value="C1" label="Atom name of atom 2"/>
+          <param name="segid3"  type="text" value="HET" label="Segid of atom 3"/>
+          <param name="resid3"  type="text" value="2" label="Resid of atom 3"/>
```

Comment:
@tsenapathi should be all ids be integers? Can you double check this?

Reply:
RESIDs should be text. They are going inside the same string with segid and atom names when we choose atoms.

Reply:
The segid , resid and atom name is used to select a specific atom in a specific residue in a specific segment. These three is used in a single string.

Reply:
but a sanitizer would be good?

Reply:
@tsenapathi what about the sanitizer? I would like to avoid that people insert `'` or other strange characters. Allowing letters and digits should be good isn't it?